lib/ava-files.js

```diff
@@ -17,7 +17,9 @@ function defaultIncludePatterns() {
 	return [
 		'test.js',
 		'test-*.js',
-		'test'
+		'test',
+		'**/__tests__/**',
```

Comment:
Are all `**/*.js` files inside `__tests__` tests? If so I think you just need to match `**/__tests__` and AVA's built in directory recursion will take over.